packages/office-ui-fabric-react/src/common/_focusBorder.scss

```diff
@@ -37,3 +37,22 @@
     outline: 1px solid $focusedBorderColor;
   }
 }
+
+@mixin pseudo-border($color) {
```

Comment:
One last request. It seems that a focus border is nothing more than pseudo-border on :focus. Can we reuse the pseudo-border inside of focus-border and get the same css in return?